files/qsub.pl

```diff
@@ -526,6 +528,12 @@ sub parse_script
     my @lines;
     @lines = split("\n", $txt) if defined $txt;
 
+    # Do not let users use #PBS -q directive
+    foreach my $line (@lines) {
+        if ($line =~ m/\s*#PBS.*?\s-q/) {
+            fatal("In SLURM no queues defined. Please do not use \"#PBS -q <QUEUE>\" directive in the jobscript");
```

Comment:
Also here, should be a warning + simply ignore whatever value is passed?

Reply:
The problem is this part handled by slurm pbs plugin. So if we issue a warning, the plugn will still pick up the queue.

Reply:
Can we strip out the `#PBS -q` line from the job script before it reaching the Slurm PBS plugin?

Reply:
Not sure it is a good idea to modify the script of the users. Right now the original script passed to sbatch, so we can still overwritten those values with command line. I cannot create a "no queue" command line option for sbatch. If we ever rewrite the wrapper, then we can do tricks (we give only the rewritten stdin (made from batch script) to slurm, then we can do that),

Reply:
I am not sure that it is a good idea to made this big change in the current wrapper. (the stdin trick)

Reply:
Fine, then error out is the next best thing, it's not a huge problem as long as the error message is clear (which it is, except for the silly typos ;-) )